e2e/pages/Table/TableModule.ts

```diff
@@ -497,6 +497,26 @@ class TableModule {
     const fspText = (await fsp.textContent())?.trim();
     expect(fspText).toBe(fspName);
   }
+
+  async selectPaByLanguage({ language }: { language: string }) {
+    await this.page.waitForSelector(TableModule.getCellValueTableRight(1, 4));
+
+    const count = await this.paCell.count();
+    for (let i = 1; i <= count; i++) {
+      const getRow = this.page.locator(TableModule.getRow(i));
+      const rowText = (await getRow.textContent())?.trim();
+      const isRequiredLanguage = rowText?.includes(language);
+
+      if (
+        (language && isRequiredLanguage) ||
+        (!language && !isRequiredLanguage)
+      ) {
```

Comment:
```suggestion
      if (isRequiredLanguage) {
```

According to how you've typed the signature, I don't think `!language` is ever true.

Reply:
Yes  you are right. Good catch